tests/src/helpers.c

```diff
@@ -295,25 +295,38 @@ void mbedtls_test_err_add_check( int high, int low,
      *                                                shhhhhhhhlllllll
      *
      * s = sign bit.
-     * h = high level error code (includes high and module error codes).
+     * h = high level error code (includes high level module ID (bits 12..14)
+     *     and module-dependent error code (bits 7..11)).
      * l = low level error code.
      */
-    if ( high > -0x1000 && high != 0 )  // high < 0001000000000000
+    if ( high > -0x1000 && high != 0 )
+    /* high < 0001000000000000
+     * No high level module ID bits are set.
+     */
     {
         mbedtls_test_fail( "'high' is not a high-level error code",
                             line, file );
     }
-    else if ( high < -0x7F80 )          // high > 0111111110000000
+    else if ( high < -0x7F80 )
+    /* high > 0111111110000000
+     * Error code is larger than the greatest high + module level error.
```

Comment:
I don't know what “greatest high + module level error” means (I don't even know how to parse it). I don't actually need an explanation here, I follow what the code is doing, but if the only way to understand the comment is to read the code, the comment isn't right.

Reply:
I think I've clarified this a bit better, hopefully it is easier to understand now.